src/main/resources/org/jenkinsci/plugins/scriptler/ScriptlerManagement/confirm-remove.js

```diff
@@ -0,0 +1,9 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const removeScriptButtons = document.querySelectorAll(".scriptler-remove-script-form");
+    removeScriptButtons.forEach(button => button.addEventListener("click", (e) => {
+        const name = e.currentTarget.getAttribute('data-name');
+        if (!confirm("Sure you want to delete [" + name + "]?")) {
+            e.preventDefault();
```

Comment:
We don't have to do anything if `confirm` is true, just let the form get submitted. If it was cancelled there we prevent the form submission. So I thought reverting the `if` makes sense here.